kahuna/public/stylesheets/main.css

```diff
@@ -308,9 +308,24 @@ a:focus {
     display: inline-block;
 }
 
-/* ==========================================================================
-   Generics
-   ========================================================================== */
+.side-padded {
+    padding: 0 10px;
+}
+
+.clickable:hover {
```

Comment:
Could this be something more descriptive - we're baking functionality with style. Perhaps topbarbutton or something?